spec/v1/Document.schema.yaml

```diff
@@ -2593,6 +2603,208 @@ definitions:
+          min: 0
+          max: "*"
+
+      partOfGroups:
+        <<: *partOfGroups
+        x-ums-reverse-relationship:
+          propertyName: agentMembers
+          description: |-
+            The Agents that are part the given Group.
+
+      version:
+        <<: *version
+
+      lastUpdate:
+        <<: *lastUpdate
+
+      visibility:
+        <<: *visibility
+
+      releaseStatus:
+        <<: *releaseStatus
+
+      disabled:
+        <<: *disabled
+
+      minSystemVersion:
+        <<: *minSystemVersion
+
+      partOfProducts:
+        <<: *partOfProducts
+        minItems: 0
+
+      responsible:
+        <<: *responsible
+
+      deprecationDate:
+        <<: *deprecationDate
+
+      sunsetDate:
+        <<: *sunsetDate
+
+      successors:
+        type: array
+        description: |-
+          The successor resource(s).
+
+          MUST be a valid reference to an ORD ID.
+
+          If the `releaseStatus` is set to `deprecated`, `successors` MUST be provided if one exists.
+          If `successors` is given, the described resource SHOULD set its `releaseStatus` to `deprecated`.
+        items:
+          type: string
+          pattern: ^([a-z0-9]+(?:[.][a-z0-9]+)*):(agent):([a-zA-Z0-9._\-]+):(v0|v[1-9][0-9]*)$
+          x-association-target:
+            - "#/definitions/Agent/ordId"
+          maxLength: 255
+
+      changelogEntries:
+        <<: *changelogEntries
+
+      policyLevel:
+        <<: *policyLevel
+
+      customPolicyLevel:
+        <<: *customPolicyLevel
+
+      policyLevels:
+        <<: *policyLevels
+
+      countries:
+        <<: *countries
+
+      lineOfBusiness:
+        <<: *lineOfBusiness
+
+      industry:
+        <<: *industry
+
+      relatedEntityTypes:
+        # NOTE: It would have been better for consistency and evolution to use the `RelatedEntityType` object
+        #       Like we do it on the Entity Type -> Entity Type relation
+        type: array
+        description: |-
+          Optional list of related EntityType Resources.
+          
+          MUST be a valid reference to an [EntityType Resource](#entity-type) ORD ID.
+        items:
+          type: string
+          pattern: ^([a-z0-9]+(?:[.][a-z0-9]+)*):(entityType):([a-zA-Z0-9._\-]+):(v0|v[1-9][0-9]*)$
+          x-association-target:
+            - "#/definitions/EntityType/ordId"
+        x-ums-reverse-relationship:
+          propertyName: agents
+          description: |-
+            Agents associated to the Entity Type.
+          min: 0
+          max: "*"
+
+        examples:
+          - ["sap.odm:entityType:WorkforcePerson:v1"]
+
+      exposedApiResources:
+        type: array
+        description: |-
+          Optional list of API Resources that expose the functionality of the agent. Typically using the A2A protocol, but other protocols are possible as well.
+
+          MUST be a valid reference to an [API Resource](#api-resource) ORD ID.
+        items:
+          type: string
```

Comment:
Are you sure about the plain ORD ID array or better include objects [{ordId: "<ord id>"}] like we do for data product output ports?

I also assume the x-ums attribute is missing?! @Fannon